javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,14 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var nums = [];
+  for (var i = 0; i < values.length; i++) {
+    if ((typeof values[i]) === 'number') {
```

Comment:
This doesn't filter out strings, it just finds numbers. What if there was other data types mixed in such as booleans?